src/components/ContinuationIn/ExtraproRegistration.vue

```diff
@@ -132,7 +132,7 @@
             sm="9"
           >
             <v-text-field
-              v-model="business.homeIdentifier"
+              v-model.trim="business.homeIdentifier"
```

Comment:
Automatically trims leading and trailing spaces. (Note: on screen, it looks like there are trailing spaces, but in the object property itself, those spaces aren't there.)

Ref: https://vuejs.org/guide/essentials/forms#trim

cc: @JazzarKarim

Reply:
Very very nice.